src/applications/yellow-ribbon/containers/SearchResults/index.jsx

```diff
@@ -209,12 +209,17 @@ export class SearchResults extends Component {
           className="va-introtext va-u-outline--none vads-u-font-size--lg vads-u-margin-top--1p5 vads-u-font-weight--normal"
           data-display-results-header
         >
-          Displaying {resultsStartNumber}
-          <span className="vads-u-visibility--screen-reader">through</span>
-          <span aria-hidden="true" role="presentation">
-            &ndash;
+          {/* eslint-disable-next-line jsx-a11y/aria-role */}
+          <span role="text">
```

Comment:
I'm not sure `text` is a role, but either way you can remove it here, it wouldn't be doing anything

Reply:
So for whatever reason, if I do not have the role there, the screen reader reads it as reported in this user story from this ticket => https://github.com/department-of-veterans-affairs/va.gov-team/issues/12060

Maybe @1Copenut or @joshkimux can explain a little more? We did a discovery on it a while back to solve the same issue in Find Forms. This is the only thing that seemed to work.

Reply:
That's super weird since it isn't listed as a valid role https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques -- if we need weird error suppression, maybe `aria-role="none"` or something really generic?

Reply:
@timwright12 , this is a fix specific to VoiceOver! [You can read about it here at axesslab](https://axesslab.com/text-splitting/). We had similar doubts too at first, but tested it and realized this was the only way we could get it to work.

Reply:
alright, fair enough